manager/integration/tests/test_scheduling.py

```diff
@@ -468,20 +467,30 @@ def test_replica_auto_balance_node_least_effort(client, volume_name):  # NOQA
     common.update_setting(client,
                           SETTING_REPLICA_AUTO_BALANCE, "least-effort")
 
-    n1, n2, n3 = client.list_node()
+    n_names = [get_self_host_id()]
+    nodes = client.list_node()
+    for node in nodes:
+        if node.name == n_names[0]:
+            continue
+        n_names.append(node.name)
+    assert len(n_names) == 3
+    assert n_names[0] == get_self_host_id()
+    n1 = client.by_id_node(n_names[0])
+    n2 = client.by_id_node(n_names[1])
+    n3 = client.by_id_node(n_names[2])
```

Comment:
There is no need to either check if the host id is in the node list, or claim a new variable `n1 n2 n3` for each node.
```
host_id = get_self_host_id()
nodes = client.list_node()
assert len(n_names) == 3

......

volume.attach(hostId=host_id)
```

Reply:
Thanks, that make sense, fixed.